datadog_checks_dev/datadog_checks/dev/tooling/templates/check/{check_name}/README.md

```diff
@@ -0,0 +1,47 @@
+# Agent Check: {check_name_cap}
+## Overview
+
+This check monitors [{check_name_cap}][1].
+
+## Setup
+
+### Installation
+
+The {check_name_cap} check is included in the [Datadog Agent][2] package, so you don't
```

Comment:
This is true of *core* Integrations, but not of *extras*; suggest hedging our default language here.